contribs/gnofaucet/go.mod

```diff
@@ -5,15 +5,13 @@ go 1.22
 toolchain go1.22.4
 
 require (
-	github.com/gnolang/faucet v0.3.2
-	github.com/gnolang/gno v0.1.0-nightly.20240627
-	github.com/stretchr/testify v1.9.0
+	github.com/gnolang/faucet v0.4.0
+	github.com/gnolang/gno v0.0.0-20241127025539-d8589b06b14c
+	github.com/stretchr/testify v1.10.0
 	go.uber.org/zap v1.27.0
 	golang.org/x/time v0.5.0
 )
 
-replace github.com/gnolang/gno => ../..
```

Comment:
should be kept.

Reply:
thinking `gnolang/faucet` should be moved into the monorepo or `gnofaucet` out of it; in any case fine.

Reply:
Move it in the monorepo.